lib/manager/npm/post-update/npm.ts

```diff
@@ -113,7 +121,7 @@ export async function generateLockFile(
     // massage to shrinkwrap if necessary
     if (
       filename === 'npm-shrinkwrap.json' &&
-      (await pathExists(upath.join(cwd, 'package-lock.json')))
+      (await findLocalSiblingOrParent(lockFileName, 'package-lock.json'))
```

Comment:
Isn't this a change in functionality? Now it will look up the directory list

Reply:
Using `findSibling` - like function was suggested by @viceice. Maybe I misunderstood?

Reply:
Yes, don't we have a `findLocalSilbling`? Maybe we can now just replace package file with lockfile in relativ package path?

Reply:
yeah, let's do that

Reply:
```suggestion
      (await localPathExists(lockFileName.replace('npm-shrinkwrap.json', 'package-lock.json'))
```